packages/grpc-js/src/channelz.ts

```diff
@@ -598,27 +585,34 @@ function GetTopChannels(
   call: ServerUnaryCall<GetTopChannelsRequest__Output, GetTopChannelsResponse>,
   callback: sendUnaryData<GetTopChannelsResponse>
 ): void {
-  const maxResults = Number.parseInt(call.request.max_results);
+  const maxResults = parseInt(call.request.max_results, 10) || 100;
```

Comment:
protofiles say there is a sane default that will be selected by the server if max_results is not provided by the request - `100` is my choice for the sane default. problem with 0 is that there will be no results returned with the current implementation. it could be a much higher number, but I think a 100 for an iteration is fine

Reply:
Especially because this is used in multiple places, it would be better to have a constant named something like `DEFAULT_MAX_REQUESTS`. Then it's clearer what the number is, and if we decide that a different value is more appropriate in the future we can change it in only one place.